pipenv/cli.py

```diff
@@ -783,11 +783,11 @@ def cleanup_procs(procs, concurrent):
                 del v['hash']
 
     # Convert the deps to pip-compatible arguments.
-    deps_list = [(d, ignore_hashes) for d in convert_deps_to_pip(deps, project, r=False, include_index=True)]
+    deps_list = [(d, ignore_hashes, blocking) for d in convert_deps_to_pip(deps, project, r=False, include_index=True)]
```

Comment:
Since you changed the variable below to `block` shouldn't this also be  `block` instead of `blocking`

Reply:
No. This `blocking` is coming from above (line 743). It's providing the regular dependencies a value of whether the install should block or not based on the `concurrent` arg passed into the function. On line 790, I'm always setting that block value to be true for vcs dependencies.

The `block` below (line 801) is unpacking the value that is added to the tuples here for use when passing into the `pip_install` call. I made it a point to name them differently so it doesn't overload the `blocking` variable and cause potential issues and confusion down the road.

Reply:
Thanks for the explanation, that makes good sense to me.